templates/bashrc.osh-template

```diff
@@ -6,7 +6,7 @@ export OSH=$HOME/.oh-my-bash
 OSH_THEME="font"
 
 # Uncomment the following line to use case-sensitive completion.
-# CASE_SENSITIVE="true"
+# export CASE_SENSITIVE="true"
```

Comment:
Under which conditions is this `export` needed?
I did some tests on my side at it works just fine without modifying the scope of this variable.